utils/chunk-tests.js

```diff
@@ -0,0 +1,152 @@
+const child_process = require('child_process');
+const path = require('path');
+
+const NUMBER_OF_CHUNKS = 5;
+const MINIMUM_PER_CHUNK = 1;
+const runnersMap = new Map([
+  // ['test-integration-dev', ['macos-latest', 'ubuntu-latest']],
```

Comment:
Should this be removed or restored?

Reply:
Since the dev integration tests currently use ava which doesn't have `--listTests` I don't think we can split those up so will remove for now

Reply:
removed in https://github.com/vercel/vercel/pull/7795/commits/a53d099852feb9019791ab523022315b64226ba4